lib/tasks/mastodon.rake

```diff
@@ -4,6 +4,258 @@ require 'optparse'
 require 'colorize'
 
 namespace :mastodon do
+  desc 'Configure the instance for production use'
+  task :setup do
+    prompt = TTY::Prompt.new
+    env    = {}
+
+    begin
+      prompt.say('Your instance is identified by its domain name. Changing it afterward will break things.')
+      env['LOCAL_DOMAIN'] = prompt.ask('Domain name:') do |q|
+        q.required true
+        q.modify :strip
+        q.validate /\A[a-z0-9\.\-]+\z/i
```

Comment:
What about internationalized domain name?

Reply:
Read the error message right below this line. This env variable needs ascii only, no unicode allowed

Reply:
Oops, sorry.
Wouldn't it make sense to have a punycode encoder, though?